### ISSUE TITLE
关于只有强标注数据集时训练策略的问题

### ISSUE THREAD

**Issue by Kev111n** [Audio-WestlakeU/ATST-SED#29]
您好，非常感谢您的开源贡献还有您对Issues的积极解答！😊

  如果我目前有大概2000~3000条音频构成的强标注数据集（事件类别为8类），我预想的训练流程是按照您的项目进行两阶段的训练，然后将模型两个阶段的损失都只保留BCE这一部分，但我担心这会像您论文中所说的出现过拟合。我想请教您，如果是您会怎么进行训练？

  祝愿您取得更多的研究成果！

---

Hello, thank you very much for your open source contribution and your active answers to issues! 😊

If I currently have a strongly labeled dataset of about 2,000 to 3,000 audios (with 8 event categories), I envision a two-stage training process according to your project, and then keep only the BCE part of the loss of the two stages of the model, but I am worried that this will cause overfitting as mentioned in your paper. I would like to ask you, how would you train if it were you?

I wish you more research results!

**Comment by SaoYear** [comment]
您好！感谢关注工作，对于您的问题，在Stage1 仅使用强标签或强弱标签同时训练的性能可以直接试一下 取最好。
在Stage 2 仅使用强标签的finetune可能需要对ATST的参数进行额外控制，如冻住更浅层的参数或使用更低的学习率。
总的来说这是一个比较细节的问题，需要具体问题具体分析。

**Comment by Kev111n** [comment]
好的，感谢您的解答！
